app/src/main/java/com/example/weathertimeapp/mvp/contracts/WeatherDetailsContracts.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.weathertimeapp.mvp.contracts
+
+import com.example.weathertimeapp.data.entities.Day
+import io.reactivex.Observable
+
+interface WeatherDetailsContracts {
```

Comment:
As common convention sets, this should be singular

Reply:
refactor to singular, thanks